lib/stat_tracker.rb

```diff
@@ -0,0 +1,14 @@
+require 'csv'
+
+class StatTracker
+    attr_reader
+ 
+    def initialize
```

Comment:
If there's nothing to do here, we shouldn't need this method (use Ruby's default)

Reply:
deleted